ui/src/flux/components/SchemaItemCategory.tsx

```diff
@@ -0,0 +1,100 @@
+// Libraries
+import React, {PureComponent} from 'react'
+
+// Components
+import MeasurementList from 'src/flux/components/MeasurementList'
+import FieldList from 'src/flux/components/FieldList'
+import TagKeyList from 'src/flux/components/TagKeyList'
+
+// Constants
+import {OpenState} from 'src/flux/constants/explorer'
+
+// Types
+import {Source, NotificationAction} from 'src/types'
+
+export enum CategoryType {
+  MEASURMENTS = 'measurements',
```

Comment:
Can these just be titled cased? I think it's pretty standard for `enum`s  (e.g. `Measurements` rather than `MEASUREMENTS`)